src/i18n/en.js

```diff
@@ -155,7 +155,7 @@ export default {
   reports_title: `Doppler | Reports`,
   signup: {
     activate_account_instructions: `* By clicking on the button from the Email, you will activate your account and you will be ready to enjoy all the benefits of Doppler.`,
-    button_signup: `create your free account`,
+    button_signup: `Sign up for free`,
```

Comment:
Here maybe it should be `Sign Up`, we have it in doppler this way.

Reply:
In this case `Sign up for free` is fine.